porter/v1/cluster_control_plane.proto

```diff
@@ -802,6 +802,32 @@ message CreateNotificationRequest {
 // CreateNotificationResponse is the response object for creating notifications
 message CreateNotificationResponse {}
 
+// UpdateServiceDeploymentStatusRequest is the request object for updating the deployment status of a service
+message UpdateServiceDeploymentStatusRequest {
+  int64 project_id = 1;
+  int64 cluster_id = 2;
```

Comment:
technically we don't need this cluster id if we have the deployment target id, but I can see why it might make certain things easier